### DISCUSSION TITLE
Oracle Cloud Infra Support

### DISCUSSION THREAD

**Discussion by venkatreddyc** [cloudforet-io/community#60] [Ideas]
Does Cloudforet support OCI

**Comment by whdalsrnt** [comment]
Hi, @venkatreddyc 

[OCI](https://github.com/cloudforet-io/plugin-oracle-cloud-service-inven-collector) integration was in progress, but development is currently suspended.
So we are looking for new contributors.

The list of currently supported plugins is as follows.
https://cloudforet.io/docs/guides/plugins/asset-inventory-collector/

Thank you.

**Reply by venkatreddyc** [reply]
@whdalsrnt , thanks for the update
